scipy/stats/tests/test_stats.py

```diff
@@ -7663,6 +7663,20 @@ def test_error_code(self):
             v_values = rng.random(size=(2, 2))
             _ = stats.wasserstein_distance(u_values, v_values)
 
+    def test_flatten(self):
+        rng = np.random.default_rng(456347456756786)
+        num_tests = 100
+        u_values_len_array = rng.integers(low=1, high=10, size=num_tests)
+        v_values_len_array = rng.integers(low=1, high=10, size=num_tests)
```

Comment:
Is it intentional to test very short arrays (e.g. length 1)?

Reply:
Yes, just in case. Is there a reason not to test them?

Reply:
No. It's good to test edge cases like that. But otherwise, I would have thought that differences would be more likely to appear for larger arrays. Consider choosing the array sizes by hand and just let the values in the arrays be random.